hs_app_timeseries/models.py

```diff
@@ -20,29 +20,17 @@ class Site(AbstractMetaDataElement):
     elevation_datum = models.CharField(max_length=50, null=True, blank=True)
     site_type = models.CharField(max_length=100, null=True, blank=True)
 
+    def __str__(self):
+        return self.site_name
+
     def __unicode__(self):
-        self.site_name
+        return unicode(str(self))
```

Comment:
Both of these are unfortunately wrong.

`CharField` values on Postgres are already unicode, decoded by Django when you fetch the object. So the proper version of your `__unicode__` method is `return self.site_name` with no need to convert.

But the `__str__` needs to encode that into an ASCII encoded string, which means handling non-ASCII characters. If this value is permitted to use non-ASCII characters (and it is, if you aren't doing anything to stop it) then you need to handle those invalid character when generating a `str`. One common option is to use one of the helper methods in `django.utils.encoding` such as the `force_bytes` function, which you might want to call with `errors="replace"` so that invalid characters are replaced with placeholder `?` characters *or* `errors="ignore"` if you want them removed in the conversion entirely.

https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.encoding.force_bytes